skins/oasis/modules/PageHeaderController.class.php

```diff
@@ -29,6 +29,7 @@ public function init() {
 		$this->displaytitle = $skinVars['displaytitle']; // if true - don't encode HTML
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)